src/main/java/gui/MainWindow.java

```diff
@@ -22,14 +22,46 @@
     private JPanel mapPanel;
     private JPanel dataPanel;
     private JLabel mapContainer;
-    private JTextField textField1;
+    private JList nodesList;
+    private JList locationsList;
+    private JList edgesList;
 
+    private DefaultListModel nodesModel;
```

Comment:
Trochę nodesModel może wybić w późniejszym etapie, gdy będziemy korzystać z bazy, zmień to lepiej na nodesListModel albo coś bardziej sugerującego, że tyczy się to wyglądu, to samo tyczy się locationsModel i edgesModel. Chyba, że źle to rozumiem.